src/handlers/xp.js

```diff
@@ -0,0 +1,103 @@
+const fetch = require('node-fetch')
+const { error, log } = require('../utils')
+const parseMyCred = require('../parser/myCred')
+const { environment } = require('../environment')
+const MessageEmbed = require('discord.js').MessageEmbed
+const NodeAddress = require('../sourcecred-utils')
+
+const filterAccount = (obj, targetUserDiscordID) => {
+  // Ignore if the target isn't a USER
+  if (obj.account.identity.subtype !== 'USER') return false
+  const discordAlias = obj.account.identity.aliases.filter(
+    alias => {
+      const parts = NodeAddress.toParts(alias.address)
+      return parts.indexOf('discord') > 0
+    })
+  if (discordAlias.length === 1) {
+    // Retrieve the Discord ID
+    const discordId = NodeAddress.toParts(discordAlias[0].address)[4]
+    if (discordId === targetUserDiscordID) {
+      return obj
+    }
+  }
+  return false
+}
+
+const scoreFind = async (message) => {
+  try {
+    // Parse message.content to get the targeted user
+    const targetParameter = parseMyCred(message.content)
+    // Remove the first characters <@! and the last > to get the Discord ID
+    const targetUserDiscordID = targetParameter.slice(3, targetParameter.length - 1)
+    if (isNaN(targetUserDiscordID)) {
+      return message.reply(
+        'You must tag a user to use this command, try `!ac xp @your-discord-username` or `!ac help` if you need help',
+      )
+    }
+
+    // Fetch accounts.json containing the xp
+    const credAccounts = await (
+      await fetch(`https://raw.githubusercontent.com/${environment('GITHUB_ACCOUNT_FILE_PATH')}`,
+      )
+    ).json()
+    const accounts = credAccounts.accounts
+    // Retrieve the Discord ID targeted
+    const userIndex = accounts.findIndex(account => filterAccount(account, targetUserDiscordID))
+    if (userIndex > -1) {
+      const userTotalCred = accounts[userIndex].totalCred
+      const lengthArray = accounts[userIndex].cred.length
+      const userWeeklyCred = accounts[userIndex].cred
```

Comment:
Instead of using findIndex, you can just use find so you dont even have to do `accounts[userIndex]` below:

```suggestion
    const user = accounts.find(account => filterAccount(account, targetUserDiscordID))
    if (user) {
      const userTotalCred = user.totalCred
      const lengthArray = user.cred.length
      const userWeeklyCred = user.cred
```

Reply:
Updated as suggested!